docs/components/DocsMenuSection.vue

```diff
@@ -26,8 +26,8 @@
   import { faAngleUp } from '@fortawesome/free-solid-svg-icons/faAngleUp'
   import { faBook } from '@fortawesome/free-solid-svg-icons/faBook'
 
-  import SlideUpDown from '@lib/components/SlideUpDown.vue'
-  import { library, default as Fa } from '@lib/components/Fa'
+  import SlideUpDown from '@/components/SlideUpDown.vue'
```

Comment:
Not for this PR but now that we have ESlint we should have a rule for import extensions :)

Reply:
Definitely :+1: